app/views/summary/unallocated.html.erb

```diff
@@ -67,7 +67,7 @@
         </td>
 
         <td aria-label="Earliest release<br/>date" class="govuk-table__cell">
-          <%= format_date(offender.earliest_release_date) %>
+          <%= format_date(offender.earliest_release_date, replacement: 'Unknown') %>
```

Comment:
I like these "Unknown" labels - would it be good to have a test covering at least one of these pages, though? 

It may just be a case of adding an expectation to `when_there_is_missing_information_spec.rb`?

Reply:
Expectation added to `when_nomis_is_missing_information_spec.rb` to cover the "Unknown" labels